src/controllers/metadata.js

```diff
@@ -32,4 +10,29 @@ const metadata = {
  *
  * @returns {void}
  */
-exports.getResource = function * () { this.body = metadata }
+module.exports = (config) => ({
+  getResource: function * () {
+    const base = config.getIn(['server', 'base_uri'])
+    const metadata = {
```

Comment:
What's the rationale for re-generating this object on every request? Since the config is immutable, you should generate it once when the outer function is called and then just return it. Probably not a huge difference performance-wise, but it also makes the code more readable imho.